ingest/src/test/java/com/microsoft/azure/kusto/ingest/E2ETest.java

```diff
@@ -221,7 +239,7 @@ void testIngestFromFile() {
             } catch (Exception ex) {
                 Assertions.fail(ex);
             }
-            assertRowCount(item.rows);
+            assertRowCount(item.rows, false);
```

Comment:
This gives us the chance to add to existing E2E tests checks of both the json approach and the KustoOperationResult approach.